src/routes/governance/current-proposal-status.tsx

```diff
@@ -14,36 +15,87 @@ export const CurrentProposalStatus = ({
     proposal,
   });
   const { t } = useTranslation();
+  const daysClosedAgo = formatDistanceToNow(
+    new Date(proposal.terms.closingDatetime),
+    { addSuffix: true }
+  );
+  const daysEnactedAgo = formatDistanceToNow(
+    new Date(proposal.terms.enactmentDatetime),
+    { addSuffix: true }
+  );
 
-  if (willPass) {
+  if (proposal.state === ProposalState.Open && willPass) {
     return (
       <span className="current-proposal-status__pass">{t("shouldPass")}</span>
     );
   }
 
   if (!participationMet) {
     return (
-      <span className="current-proposal-status__fail">
-        {t("participationNotMet")}
-      </span>
+      <>
+        <span>{t("voteFailedReason")}</span>
+        <span className="current-proposal-status__fail">
+          {t("participationNotMet")}
+        </span>
+        <span>&nbsp;{daysClosedAgo}.</span>
+      </>
     );
   }
 
   if (!majorityMet) {
     return (
-      <span className="current-proposal-status__fail">
-        {t("majorityNotMet")}
-      </span>
+      <>
+        <span>{t("voteFailedReason")}</span>
+        <span className="current-proposal-status__fail">
+          {t("majorityNotMet")}
+        </span>
+        <span>&nbsp;{daysClosedAgo}.</span>
+      </>
     );
   }
 
-  if (proposal.state === ProposalState.Failed) {
-    return <span className="current-proposal-status__fail">{t("failed")}</span>;
```

Comment:
I don't think this translation is used anywhere else (also `passed`) and can be removed. Could you verify any translation changes and remove any which are unused

Reply:
done